spec/requests/review_pages_spec.rb

```diff
@@ -255,7 +255,7 @@
          review.reload
          review.feedback_deadline.to_s(:short_date).should == Date.parse(new_deadline).to_s(:short_date)
          page.should have_selector('.flash-success',
-          text: 'Review was updated. Your coach and invitees have been notified of the new deadline.')
+          text: 'Review was updated. Your invitees have been notified of the new feedback deadline.')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.